src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -519,6 +521,13 @@ public static String applyModifiers(String label, List<String> parts, int offset
                     }
                     resultingLabel = abbreviateSB.toString();
 
+                } else if ("camel".equals(modifier)) {
```

Comment:
Turns out that the (useless) camel case formatter is actually already available as `CapitalizeFormatter`, then there is the `SentenceFormatter` left and that I can add for completeness (`ProtectedTermsFormatter` really doesn't make sense here...).